code/game/objects/items/stacks/sheets/leather.dm

```diff
@@ -190,6 +190,7 @@ GLOBAL_LIST_INIT(leather_recipes, list ( \
 	new/datum/stack_recipe("saddle", /obj/item/saddle, 5), \
 	//SKYRAT EDIT ADDITION BEGIN
 	new/datum/stack_recipe("gear harness", /obj/item/clothing/under/misc/gear_harness, 6), \
+	new/datum/stack_recipe("loincloth", /obj/item/clothing/under/costume/loincloth, 2), \
```

Comment:
these ones are fine but could use more shading and detailing on the sprites
![chrome_pm7T7rB3Jn](https://user-images.githubusercontent.com/4081722/159590076-98e6b098-0810-4004-94ef-8faff75d9243.png)